services/src/main/scala/cromwell/pipeline/service/ServiceModule.scala

```diff
@@ -16,7 +16,7 @@ class ServiceModule(
   implicit executionContext: ExecutionContext
 ) {
   lazy val authService: AuthService =
-    new AuthService(datastorageModule.userRepository, authModule.authUtils)
+    new AuthService(new UserService(datastorageModule.userRepository), authModule.authUtils)
```

Comment:
Sorry for not noticing it earlier, but why do we create another instance of `UserService` here? Can't we use `userService` defined on the line below?

Reply:
Thanks! I did not notice

Reply:
@myazinn done